tools/lint-all

```diff
@@ -83,9 +83,9 @@ def check_pep8(files):
     failed = False
     ignored_rules = [
         'E402', 'E501', 'W503', 'E711', 'E201', 'E203', 'E202', 'E128', 'E226', 'E124', 'E125',
-        'E126', 'E127', 'E121', 'E122', 'E123', 'E266', 'E265', 'E261', 'E701', 'E221', 'E303',
-        'E241', 'E712', 'E702', 'E401', 'E115', 'E114', 'E111', 'E222', 'E731', 'E302', 'E129',
```

Comment:
The only change we want here is to remove the `'E127'`.
We specifically don't want to reflow the lines around it. So the line with the `'E127'` should be slightly shorter than the rest and none of the other two lines should show additions or removals.

We want to minimize the change to the surrounding lines since a bunch of other people are also modifying this section in other tasks. The problem is that git uses a line based approach for diffs which means if more lines are modified there will be more merge/rebase conflicts.